pkg/kine/logstructured/sqllog/sql.go

```diff
@@ -124,17 +124,25 @@ func (s *SQLLog) DoCompact(ctx context.Context) error {
 }
 
 func (s *SQLLog) compactor(ctx context.Context, nextEnd int64) (int64, error) {
+	_, span := tracer.Start(ctx, "sqllog.compactor")
+	defer span.End()
+	span.AddEvent(fmt.Sprintf("nextEnd: %d", nextEnd))
```

Comment:
Could we use attributes here instead of formatting a string? I see that in `limitedserver` we are doing that